src/components/booklist.js

```diff
@@ -39,53 +26,57 @@ export default function BookList() {
 		}
 	}
 
+	const submit = id => {
+		confirmAlert({
+			message: 'Do you really want to delete this book?',
+			buttons: [
+				{
+					label: 'Yes',
+					onClick: () => deleteBook(id),
+				},
+				{
+					label: 'No',
+				},
+			],
+		});
+	};
+
 	return (
 		<>
-			<form
-				onSubmit={async event => {
-					event.preventDefault();
-					const formData = new FormData(event.target);
-					const formValues = Object.fromEntries(formData);
-					try {
-						const response = await fetch('/api/books', {
-							method: 'POST',
-							body: JSON.stringify(formValues),
-							headers: {'Content-Type': 'application/json'},
-						});
-						if (response.ok) {
-							const data = await response.json();
-							console.log('returned book:', data);
-							setBooks([...books, data]);
-						} else {
-							throw new Error('post failed');
-						}
-					} catch (error) {
-						console.log(error);
-					}
-				}}
-			>
-				<label htmlFor="title">Title: </label>
-				<input type="text" name="title" placeholder="Moby Dick" required></input>
-				<label htmlFor="author">Author: </label>
-				<input type="text" name="author" placeholder="Herman Melville" required></input>
-				<label htmlFor="rating">Rating: </label>
-				<input type="number" min="1" max="10" name="rating" required></input>
-				<button type="submit">Add</button>
-			</form>
-			<ul>
-				{books.map(book => (
-					<li key={book._id}>
-						{book.title} by {book.author} Rating: {book.rating}{' '}
-						<button
-							onClick={() => {
-								deleteBook(book._id);
-							}}
-						>
-							Delete
-						</button>
-					</li>
-				))}
-			</ul>
+			<Container>
+				<StyledList>
+					{books?.map(book => (
+						<li key={book._id}>
+							<Card>
+								<div>
+									<p>
+										{book.title} by {book.author}
+									</p>
+									<p>
+										Rated: {book.rating}
+										{'/10'}
+									</p>
+								</div>
+
+								<Button onClick={() => submit(book._id)}>Delete</Button>
+							</Card>
+						</li>
+					))}
+				</StyledList>
+			</Container>
 		</>
 	);
 }
+
+const Container = styled.div`
+	padding: 10px;
+	gap: 5px;
```

Comment:
As far as I know `gap` only works for flex container. Please try out if something changes if you remove this declaration. If not, then you do not need `gap`.